lib/settings/defaults.js

```diff
@@ -248,6 +266,14 @@ module.exports = {
     on_failure: true
   },
 
+  // HTML report options that will be use by html report
+  html_reporter: {
+    // retain html report across test runs
+    retain_report: false,
```

Comment:
Do we really need a flag? What would be the best scenario is that we append the timestamp in the `index.html` file which will be unique, as well as let us know which file has the latest report

Reply:
We can't change `index.html` as it would be used in different CI pipelines to host the HTML page. In future we would need to maintain a single index.html containing all the results from previous test runs. We need a flag to preserve the default behavior and not create breaking changes for those using HTML reporter.

Reply:
I think we don't need a flag. The user can just define a `folder_format` function that will be `undefined` by default, in which case the current behaviour stays. 

We should probably use `reporter` as the settings name, because we might have other options there. We can also in the future move the `output_folder` setting in it and have something like:
```
reporter: {
  output_folder: '',
  folder_format: function() {}
}
```

Reply:
or it can be a function that returns the default value, as you've implemented here.